### PR TITLE
[DOC-269] added prepare the target database in bulk data load

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-voyager/migrate/bulk-data-load/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md [hunk 1]
```diff
@@ -18,6 +18,49 @@ The following page describes the steps to import data in CSV or TEXT format from
 
 * Before you perform a bulk load, in your target YugabyteDB database, create the schema of the tables into which the data in the flat files will be imported.
 
+### Prepare the target database
+
+Prepare your target YugabyteDB database cluster by creating a database, and a user for your cluster.
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md:23]: Prepare your target YugabyteDB database cluster by creating a user for your cluster.

File: docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md [hunk 2]
```diff
@@ -18,6 +18,49 @@ The following page describes the steps to import data in CSV or TEXT format from
...
+
+#### Create the target database
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md:25]: Remove this section of creating a database as it is not required for import data file, this is prerequisite.

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md [hunk 3]
```diff
@@ -18,6 +18,41 @@ The following page describes the steps to import data in CSV or TEXT format from
 
 * Before you perform a bulk load, in your target YugabyteDB database, create the schema of the tables into which the data in the flat files will be imported.
 
+## Prepare the target database
+
+Prepare your target YugabyteDB database cluster by creating a user for your cluster.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md:23]:
```suggestion
Prepare your target YugabyteDB database cluster by creating a user.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**